src/main/java/seedu/address/logic/parser/ImportCommandParser.java

```diff
@@ -26,8 +26,17 @@ public ImportCommand parse(String args) throws ParseException {
         }
 
         String url = argMultimap.getValue(PREFIX_TIMETABLE_URL).get();
-        TimetableData data = TimetableUrlParser.parseTimetableUrl(url);
-        return new ImportCommand(data);
+
+        if (!TimetableUrlParser.isValidUrl(url)) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, ImportCommand.MESSAGE_USAGE));
+        }
```

Comment:
Maybe instead of using the standard invalid command format message, telling the user that the URL is invalid may be a more meaningful error message :)

Reply:
Added in some new messages